Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.2'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
-
+gem 'omniauth-github'
```

Comment:
I was having trouble getting it to work, so I did it the long way and plan to figure it out the easy way (for github). Is it okay if I were to leave it done by hand?